cmd/minishift/cmd/start.go

```diff
@@ -439,35 +353,35 @@ func setDefaultRoutingPrefix(ip string) {
 	}
 }
 
-func validateOpenshiftVersion() {
-	if viper.IsSet(startFlags.OpenshiftVersion.Name) {
-		if !minishiftUtil.ValidateOpenshiftMinVersion(viper.GetString(startFlags.OpenshiftVersion.Name), constants.MinOpenshiftSuportedVersion) {
-			fmt.Printf("Minishift does not support Openshift version %s ."+
-				"You need to use a version >=%s\n", viper.GetString(startFlags.OpenshiftVersion.Name),
-				constants.MinOpenshiftSuportedVersion)
-			atexit.Exit(1)
-		}
+func ensureNotRunning(client *libmachine.Client, machineName string) {
```

Comment:
@hferentschik related to an earlier comment: https://github.com/minishift/minishift/pull/962#discussion_r119299583